ml_service/pipelines/build-model-container-image.Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+pipeline {
+    agent { label 'master' }
+    environment {
+        ML_IMAGE_FOLDER         = 'imagefiles'
+        IMAGE_NAME              = 'mlmodelimage'
+        MODEL_NAME = 'diabetes_regression_model.pkl'
+        MODEL_VERSION = '1'
+        SCORE_SCRIPT = 'scoring/score.py'
+        RESOURCE_GROUP = "${RESOURCE_GROUP}"
+        WORKSPACE_NAME = "${WORKSPACE_NAME}"
+        ML_CONTAINER_REGISTRY   = "${ML_CONTAINER_REGISTRY}"
+    }
+    stages {
+        stage('initialize') {
+            steps {
+                echo 'Remove the previous one!'
+            }
+            post {
+                always {
+                    deleteDir() /* clean up our workspace */
+                }
+            }
+        }
+        stage('generate_dockerfile') {
+            steps {
+                echo "Hello build ${env.BUILD_ID}"
+                /*checkout scm*/
+                checkout([$class: 'GitSCM', branches: [[name: '*/ml_model_uc76']],
+                    userRemoteConfigs: [[url: 'https://github.com/Merlion-Crew/MLOpsPython.git/']]])
+
+                azureCLI commands: [[exportVariablesString: '/id|SUBSCRIPTION_ID', script: "az account show"]], principalCredentialId: "${AZURE_SP}"
+                
+                sh '''#!/bin/bash -ex
+                    source /home/azureuser/anaconda3/bin/activate mlopspython_ci
+                    python3 -m ml_service.util.create_scoring_image
+                '''
+            }
+        }
+        stage('build_and_push') {
+            steps {
+                echo "Build docker images"
+
+                sh '''#!/bin/bash -ex
+                    az acr login --name $ML_CONTAINER_REGISTRY
+                    docker build -t $NEXUS_DOCKER_REGISTRY_URL/$IMAGE_NAME:$BUILD_ID ./diabetes_regression/scoring/$ML_IMAGE_FOLDER/ 
```

Comment:
I wonder if it wouldn't be better to download the image from ACR and then push. This would create the image twice (once from the [Model.package](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.model.model?view=azure-ml-py#package-workspace--models--inference-config-none--generate-dockerfile-false--image-name-none--image-label-none-) ( call in the python script) and then from here.